frontend/src/containers/map/content/map/layer-manager/index.tsx

```diff
@@ -19,18 +19,25 @@ const LayerManager = ({}: { cursor: CustomMapProps['cursor'] }) => {
   const [layersSettings] = useSyncMapLayerSettings();
 
   const getSettings = useCallback(
-    (l: number) => ({
-      ...(layersSettings[l] ?? { opacity: 1, visibility: true, expand: true }),
+    (slug: string) => ({
+      ...(layersSettings[slug] ?? { opacity: 1, visibility: true, expand: true }),
       zoom,
     }),
     [layersSettings, zoom]
   );
 
   const layerManagerItems = useMemo(
     () =>
-      activeLayers.map((l, i) => {
```

Comment:
glad to see I'm not the only one who's one character variable habit needs breaking. 😅